src/Appwrite/Platform/Modules/Projects/Http/DevKeys/Get.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Appwrite\Platform\Modules\Projects\Http\DevKeys;
+
+use Appwrite\Extend\Exception;
+use Appwrite\SDK\AuthType;
+use Appwrite\SDK\ContentType;
+use Appwrite\SDK\Method;
+use Appwrite\SDK\Response as SDKResponse;
+use Appwrite\Utopia\Response;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\UID;
+use Utopia\Platform\Action;
+use Utopia\Platform\Scope\HTTP;
+
+class Get extends Action
+{
+    use HTTP;
+    public static function getName()
+    {
+        return 'getDevKey';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->setHttpMethod(Action::HTTP_REQUEST_METHOD_GET)
+            ->setHttpPath('/v1/projects/:projectId/dev-keys/:keyId')
+            ->desc('Get dev key')
+            ->groups(['api', 'projects'])
+            ->label('scope', 'projects.read')
+            ->label('sdk', new Method(
+                namespace: 'projects',
+                name: 'getDevKey',
+                description: <<<EOT
+                Get a project\'s dev key by its unique ID. Dev keys are project specific and allow you to bypass rate limits and get better error logging during development.
+                EOT,
+                auth: [AuthType::ADMIN],
+                responses: [
+                    new SDKResponse(
+                        code: Response::STATUS_CODE_OK,
+                        model: Response::MODEL_DEV_KEY
+                    )
+                ],
+                contentType: ContentType::JSON
+            ))
+            ->param('projectId', '', new UID(), 'Project unique ID.')
+            ->param('keyId', '', new UID(), 'Key unique ID.')
+            ->inject('response')
+            ->inject('dbForPlatform')
+            ->callback(fn ($projectId, $keyId, $response, $dbForPlatform) => $this->action($projectId, $keyId, $response, $dbForPlatform));
+    }
+
+    public function action(string $projectId, string $keyId, Response $response, Database $dbForPlatform)
+    {
+
+        $project = $dbForPlatform->getDocument('projects', $projectId);
+
+        if ($project->isEmpty()) {
+            throw new Exception(Exception::PROJECT_NOT_FOUND);
+        }
+
+        $key = $dbForPlatform->findOne('devKeys', [
+            Query::equal('$id', [$keyId]),
+            Query::equal('projectInternalId', [$project->getInternalId()]),
+        ]);
```

Comment:
Same comment as in Update.php, let's getDocument and do if to ensure projectInternalId